includes/AssignPostTable.php

```diff
@@ -106,7 +106,7 @@
 
 while ($row = mysqli_fetch_row($res))
 {
-	if( ! empty($_POST[$row[0]]) )
+	if( isset($_POST[$row[0]]) )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after IF keyword; 0 found
* Expected 0 spaces after opening bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 5

<!-- Comment by [SideCI](https://sideci.com) -->